docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,361 @@ card(
       {
         name: 'disabled',
         type: 'boolean',
+        description: `Used to disable the entire SelectList. See the [states](#States) variant to learn more.`,
         defaultValue: 'false',
       },
       {
         name: 'errorMessage',
         type: 'string',
-        href: 'exampleWithError',
+        description: `Used to communicate error information to the user. Be sure to localize the text. See the [states](#States) variant to learn more.`,
       },
       {
         name: 'helperText',
         type: 'string',
-        description: 'More information about how to complete the form field',
-        href: 'helperText',
+        description: `Used to provide more information about the form field. Be sure to localize the text. See the [helper text](#Helper-text) variant to learn more.`,
       },
       {
         name: 'id',
         type: 'string',
+        description:
+          'A unique identifier to connect the underlying `<select>` with the associated label',
```

Comment:
nit (the sort that shouldn't be a blocker, but can be addressed if you're making other changes): there's some inconsistency here with finals periods. I believe we always want to include them, even if there's only one sentence

Reply:
Done!